.gitattributes

```diff
@@ -20,278 +20,3 @@ databricks/sdk/service/sharing.py linguist-generated=true
 databricks/sdk/service/sql.py linguist-generated=true
 databricks/sdk/service/vectorsearch.py linguist-generated=true
 databricks/sdk/service/workspace.py linguist-generated=true
-examples/account/billable_usage/download_usage_download.py linguist-generated=true
```

Comment:
This is the same issue we had for other SDKs for which we have created a backlog ticket to track.